takwimu/templates/takwimu/_includes/navbar/_dropdown.html

```diff
@@ -11,12 +11,17 @@
   <div class="container">
     <div class="row">
       <div class="col-3">
-        <h2 class="font-weight-bold">{% block dropdown_title %}Some title{% endblock %}</h2>
-        <p class="font-weight-light">{% block dropdown_desc %}Lorem ipsum{% endblock %}</p>
+        <h2 class="font-weight-bold text-right pr-3">{% block dropdown_title %}Some title{% endblock %}</h2>
+        <p class="font-weight-light text-right pr-3" style="line-height:1.61;font-weight:100;">{% block dropdown_desc %}Lorem ipsum{% endblock %}</p>
```

Comment:
Please remove the style attribute on this because:
1. The line-height here is unimportant.
2. The font-weight-light is all that is needed.